docs/photoniq/fingerprinting/integrate-fingerprint.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 25
+title: Integrate Fingerprint
+---
+
+For maximum accuracy and best results, the PhotonIQ Fingerprint application and cookies must both be integrated as first parties. This means that their URLs match the website that they are attached to.
+
+## First-party Integration Benefits
+
+For the most accurate identification, the browser must consider the Fingerprint script as “first party”. This accrues several benefits:
+
+- Significant increase in accuracy in browsers with strict privacy features such as Safari or Firefox.
+- First-party cookies can live longer in the browser and extend the lifetime of visitor IDs. For a more detailed explanation of how this affects cookies, refer to [First- and Third-Party Cookies](first-third-party-cookie.md).
+- Ad blockers will not block the Fingerprint agent from loading. Attempts to connect to an external URL will be stopped by most ad blockers while attempts to connect to the same site URL will be allowed.
+- Ad blockers will not block Fingerprint identification requests, because they are sent to the specified path or subdomain that belongs to the same site.
+
+## Integration Options
+
+There are two strategies for integrating Fingerprint as a first party:
+
+- (Recommended) Set up a cloud proxy using a tool such as Akamai, Cloudflare, or Nginx.
+- Set up a subdomain
+
+Either strategy requires coordination with your Macrometa partner, because there needs to be an administrator on both sides in order to map the URLs for first-party integration.
+
+:::note
+With the release of Safari 16.4, cookie lifetime in Safari is reduced to 7 days when using the custom subdomain setup. We recommend using a cloud proxy integration to have cookies last up to one year in Safari.
+:::
+
+### Cloud Proxy Setup
+
+For first-party integration, you must ensure all traffic to the PhotonIQ server comes from your company's domain.  This is accomplished by redirecting all traffic to the server using a cloud proxy such as Akamai.
```

Comment:
`This is accomplished by redirecting forwarding traffic to the server using a cloud proxy such as Akamai`